lib/workers/repository/update/branch/index.ts

```diff
@@ -73,17 +77,38 @@ async function deleteBranchSilently(branchName: string): Promise<void> {
 }
 
 export interface ProcessBranchResult {
+  configAndManagersHash: string;
   branchExists: boolean;
   prBlockedBy?: PrBlockedBy;
   prNo?: number;
   result: BranchResult;
 }
 
+function doesBranchNeedUpdate(
```

Comment:
```suggestion
function canSkipBranchUpdateCheck(
```

I think this makes it easier to understand